pkg/aws/commandbuilder/helper/roles/roles.go

```diff
@@ -52,6 +52,7 @@ func ManualCommandsForMissingOperatorRole(input ManualCommandsForMissingOperator
 
 type ManualCommandsForUpgradeOperatorRolePolicyInput struct {
 	HasPolicy                bool
+	PolicyAttached           bool
```

Comment:
Can policyAttached by renamed to something like
RequiredRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. 😢